admiral/debug_functions.sqf

```diff
@@ -287,6 +287,88 @@ adm_debug_fnc_updateLineMarker = {
     DEBUG("admiral.debug",FMT_3("Updated line marker '%1' position between '%2' and '%3'.",_markerName,_posFrom,_posTo));
 };
 
+adm_debug_fnc_createDebugCounterMarker = {
+    FUN_ARGS_3(_side,_type,_xPos);
+    
+    PVT_2(_marker,_markerType);
+    _markerType = if (_type != "total") then {
```

Comment:
There is no need for this `if`. You can just pass in the `_markerType` from the caller.